ansible/scripts/deploy-cluster.sh

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-inventory=${INVENTORY:-inventory}
+. ./init.sh
 
-ansible-playbook -i ${inventory} deploy-cluster.yml $@
+inventory=${INVENTORY:-${INVENTORY_DIR}/inventory}
```

Comment:
Do we really need these scripts? They are just hiding what ansible-playbook offers natively.